### PR TITLE
[DOCS] Added link to ordered selections

### LINKED ISSUE #3583: add link to `ordered-selections`
The page on [selection commands]( https://docs.mdanalysis.org/2.1.0-dev0/documentation_pages/selections.html?highlight=ordered%20selections#selection-commands) contains a note about `ordered-selections`, which shows very far from the corresponding section.

Task: add a link in the note to the section on `ordered-selections`.

### PR DESCRIPTION
**Pull request by umak1106**
Fixes #3583 

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

File: package/doc/sphinx/source/documentation_pages/selections.rst [hunk 1]
```diff
@@ -54,6 +54,7 @@ packages with the help of :ref:`Selection exporters`.
         True
 
 
+    Navigate to :ref:`ordered-selections-label`
```
Comment (micaela-matta) [inline package/doc/sphinx/source/documentation_pages/selections.rst]: Before we merge, can you change this to say something like: "for further details, see `ordered-selections` "?

**Review by micaela-matta — CHANGES_REQUESTED** [review]

File: package/doc/sphinx/source/documentation_pages/selections.rst (hunk 1 above)
Comment (umak1106) [reply package/doc/sphinx/source/documentation_pages/selections.rst]: done

File: package/doc/sphinx/source/documentation_pages/selections.rst [hunk 2]
```diff
@@ -54,6 +54,7 @@ packages with the help of :ref:`Selection exporters`.
         True
 
 
+    For further details , see :ref:`ordered-selections-label`
```
Comment (ojeda-e) [inline package/doc/sphinx/source/documentation_pages/selections.rst]: There is an extra space before the comma here that can be removed. Could you also add a trailing period at the end of this line as well?

```suggestion
    For further details on ordered selections, see :ref:`ordered-selections-label`.
```

Comment (umak1106) [reply package/doc/sphinx/source/documentation_pages/selections.rst]: done

Comment (umak1106) [reply package/doc/sphinx/source/documentation_pages/selections.rst]: done

**Comment by umak1106** [comment]
Hello mentors @ojeda-e and @micaela-matta ,
In case something is still wrong with my PR could you please let me know so I can work on it .

**Review by ojeda-e — CHANGES_REQUESTED** [review]
Thanks @umak1106. 
I think the main goal of this PR was addressed. However, I have a couple of general comments. 

When submitting Pull Requests (PR) in MDAnalysis, it is useful and also a good practice to reference the issue (as you did) and summarize the changes in the PR description. In MDAnalysis, the PR description includes items that should be checked according to the submitted changes. It is useful for contributors and reviewers since it helps to identify the changes and what is still missing in the PR.

As you can see [here](https://github.com/MDAnalysis/mdanalysis/pull/3589#issue-1188568001), we have four items to check: 

- [ ] Tests?
- [ ]  Docs?
- [ ] CHANGELOG updated?
- [ ] issue raised/referenced?

The idea is that every time we submit a PR, we check the corresponding boxes. In your case, the PR doesn't include tests, but the Docs were changed and you did reference the issue. With the change you are submitting, you should also update the CHANGELOG file by adding your GitHub handle to the list of contributors. **Edit:** And the AUTHORS file by adding your name at the end of the list. 

Would you please update the ~~CHANGELOG~~ CHANGELOG and AUTHORS files, and check the correct boxes in the PR description? 

Additionally, if you haven't done so, please introduce yourself on the MDAnalysis mailing list and let us know the PR number together with your name and GitHub username.

**Comment by ojeda-e** [comment]
@umak1106 we're almost done here :)
There is a conflict in the CHANGELOG. Please add your name using the most recent version of the file.

**Review by ojeda-e — CHANGES_REQUESTED** [review]
Hi @umak1106,
The [checks](https://github.com/MDAnalysis/mdanalysis/pull/3589/checks) show there is an issue with the docs and they are not building properly. 

We need all tests to pass before approving changes.

File: package/doc/sphinx/source/documentation_pages/selections.rst [hunk 3]
```diff
@@ -54,6 +54,7 @@ packages with the help of :ref:`Selection exporters`.
         True
 
 
+    For further details on ordered selections, see :ref:`ordered-selections-label`.
```
Comment (ojeda-e) [inline package/doc/sphinx/source/documentation_pages/selections.rst:56]: [This](https://github.com/MDAnalysis/mdanalysis/runs/5807270775?check_suite_focus=true#step:5:1434) message may suggest a blank line after.

Comment (umak1106) [reply package/doc/sphinx/source/documentation_pages/selections.rst:56]: codecov/project failed . But the docs are built now I guess.

**Comment by orbeckst** [comment]
There are some tests failing due to not being able to upload the code coverage (namely main_tests ubuntu-latest 3.8 & 3.9). These failures are unrelated to anything in the PR itself and the tests themselves passed. @ojeda-e , you wouldn't need to wait for those to pass. If all your comments were addressed, you can approve the PR and **squash-merge**.

When you squash-merge, edit the metadata before hitting the final submission button
* write a concise subject line; in this case the PR title *Added link to ordered selections* is already good, I'd just add "docs" somewhere, e.g. "[DOCS] Added link to ordered selections" or "Added link to ordered selections to docs".
* combine log messages from all the commits into something readable: 
  * first line: *fix #3583*
  * then use the PR top comment (especially as you already requested a nice summary in your PR review) to summarize the changes
  * use the individual commit messages as necessary but don't be afraid to delete them if they clutter 
  *  definitely remove duplicates or messages that are clearly intermediates ("tried X", "reverted Y"): ultimately, think of what will be useful for later readers

If you have any questions about the process just ask here or on Discord.

**Review by ojeda-e — APPROVED** [review]
Thanks for your contribution @umak1106! I am approving this PR.

Thanks to @orbeckst for the pointers.

**Review by micaela-matta — APPROVED** [review]

**Comment by orbeckst** [comment]
Congratulations @umak1106  , first PR merged 🥳

**Pull request merged**